src/manager.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-use-before-define */
```

Comment:
Circular dependency.

Reply:
Could you please explain more? I believe that comment will not be circular dependency?

Reply:
the [file](https://github.com/wechaty/matrix-appservice-wechaty/blob/8c32cf297d12325248105b2078a6d8469a7d45a8/src/manager.ts) defines two types, which refer to each other in the data structure.
```
export interface Managers {
  [manager: string]: Manager
}

export abstract class Manager {
  public abstract teamManager (managers: Managers): void
}
```

Reply:
This is OK because the `interface` is a typing definition and it will be removed at the runtime.